src/pages/Thanks/ThanksPage.vue

```diff
@@ -450,7 +450,7 @@ export default {
 		userHasDepositBefore(hasDepositBefore);
 
 		const totalLoans = data?.my?.loans?.totalCount ?? 0;
-		const isFirstLoan = this.loans.length && totalLoans === this.loans.length;
+		const isFirstLoan = this.loans.length && (totalLoans === this.loans.length || this.isGuest);
```

Comment:
This is affecting all guest checkout, not just this experiment. Is that expected? And is this because `data?.my?.loans?.totalCount` is `undefined` for guests?

Reply:
Yeah it we only want this to affect the experiment, a small adjustment is needed.